tests/scripts/final_rc_checks/final_rc_checks.py

```diff
@@ -0,0 +1,41 @@
+import requests
+import json
+import argparse
+
+# Set parser variable to reduce redundancy
+parser = argparse.ArgumentParser()
+
+# -fqdn FQDN -u Username -p Password -api API_Bearer_token
```

Comment:
Tried running locally with `-u` and `-p` but I receive error `final_rc_checks.py: error: unrecognized arguments: -u ... -p ...`. In the comment, can you remove `-u` and `-p`? As specifying only `fqdn` and `-api` worked fine for me.